### DISCUSSION TITLE
Make World Boss loot BoE with the release of Black Temple

### DISCUSSION THREAD

**Discussion by Spoon722** [chromiecraft/chromiecraft#7231] [Suggestions]
With patch 2.4 (Sunwell patch) world boss drops became BoE.
This would be nice for alts or just to make a quick buck out of it implying no-one in the group needs the items.
https://www.wowhead.com/patchnotes=2.4.0

**Comment by Cooties-cc** [comment]
It's not a bad idea, although I think the intentions of the BoE conversion was to treat them more as catchup gear. From a personal perspective, the gun for my class is still considered BiS going into BT gear pool. I'm not fully sure how much of the other gear from the 2 bosses remains as BiS for players, but if the sentiment is to have earlier access as a means to treat them as catchup gear, then perhaps a better timeframe would to make them BoE alongside the ZA release. This raid was defacto a catchup raid for players trying to enter and do most of T6 content. Converting them to BoEs would increase the catchup gear pool for new/alt players, and not appear as if it was out of line for accessibility. You could also advocate for primal nethers to also be BoE at this stage, instead of with SWP patch 2.4, which would further assist in overall gear catchup pooling.
